resolver/src/resolver_future.rs

```diff
@@ -97,7 +97,7 @@ impl ResolverFuture {
 
     /// Constructs a new Resolver with the system configuration.
     ///
-    /// This will read the systems `/etc/cresolv.conf` on Unix OSes.
+    /// This will read the systems `/etc/resolv.conf` on Unix OSes.
```

Comment:
This also supports 64bit Windows by reading from the Registry. mind adding that to this comment?